### DISCUSSION TITLE
Microsoft PowerToys (Pending developer)

### DISCUSSION THREAD

**Discussion by jonnywright** [selfhst/icons#82] [New Icon Request]
https://github.com/microsoft/PowerToys

Granted, this could be more complicated than it first looks, as people may also want icons for specific modules. I know that the icons are all available already in high quality in the PowerToys repo, but it involves some digging and would be nice to have in the already very comprehensive, and easy to navigate, selfh.st icon app.

**Comment by shollyethan** [comment]
Hey, just wanted to let you know this is on the list - I'll report back here when they've all been added.

**Comment by shollyethan** [comment]
I've had trouble finding icons for the remainder of the tools that are high quality enough to add to the collection. I'm closing this discussion for now. Thanks!
